pagefaultgames/pokerogue
# [Feature][UI] Save Preview

## What are the changes the user will see?
Users can use Button.RIGHT to see more information about their runs. 

## Why am I making these changes?
I thought it would be neat. 

## What are the changes from a developer perspective?
Run Info uses an enum to determine its purpose and chooses to display/not display certain information. 
Trainer Parties are not displayed in this mode. Only the trainer sprites are.
New function added to ME Utils to facilitate getting ME locale strings without having to generate an entire ME.

### Screenshots/Videos
Full UX

https://github.com/user-attachments/assets/4e7b8fde-5dce-44db-9de5-4ef35496bda1



Save Previews:
<details>
  <summary>Mystery Encounters</summary>

![image](https://github.com/user-attachments/assets/827dfbde-a525-48e0-9d02-53846255a39a)

</details>
<details>
  <summary>Trainer Double</summary>

![image](https://github.com/user-attachments/assets/683270f2-1f99-4575-957a-e5dfa1dad4d5)

</details>
<details>
  <summary>Trainer Single</summary>

![image](https://github.com/user-attachments/assets/db886234-4087-49bf-bdd9-790a8b86034b)

</details>
<details>
  <summary>Wild Double</summary>

![image](https://github.com/user-attachments/assets/9d9e86c8-dda8-4070-b976-3322058aece9)

</details>

---

<details>
  <summary>Got the scrolling to work now</summary>

https://github.com/user-attachments/assets/3d9591ee-e371-49f9-92e0-9374b7b56538

</details>

## How to test the changes?
Press right on any session save slot. 
Check in a variety of situations / languages -> Important to see if it works for all trainers :) 

## Checklist
